pages/index/index_layout.py

```diff
@@ -20,7 +20,7 @@
                                 dbc.DropdownMenu(
                                     children=[
                                         dbc.DropdownMenuItem(
-                                            "Refresh",
+                                            "Refresh Groups",
```

Comment:
Move refresh groups to its own tab (example "chaoss" on the other side) for user can see it without clicking in

Reply:
as in above comment, should be a redesign PR, not this one

Reply:
@JamesKunstle only necessary change to do this is remove the refresh groups as a dropdown menu item and add 
dbc.NavItem(
                                dbc.NavLink(
                                        "Refresh Groups",
                                        id="refresh-button",
                                    ),
                            ),

after(or before) the drop downmenu

Reply:
@JamesKunstle I dont feel like its necessary to do a whole different pr for such a minor change. Let me know what you think after seeing the solution

Reply:
@cdolfi I'm happy to do that but do we want to keep the dropdown if the only thing in there is the logout button? that's why I'm suggesting a redesign PR

Reply:
@JamesKunstle My first instinct is to take it out completely but for this PR Im fine with just the refresh button being on the outside. I just think it is important to have it visible before going live with the log in as I think most/all users wont intuitively look for it and think the app is broken. Code to remove completely:
```
dbc.NavItem(
      dbc.NavLink(
              "Refresh Groups",
              id="refresh-button",
          ),
  ),
dbc.NavItem(
    dbc.NavLink(
            "Log out",
            id="logout-button",
        ),
), 
```

Reply:
okay done